README.md

```diff
@@ -3,45 +3,50 @@
 shared consumer for consuming from event-bridge and emitting to snowplow.
 ![Architecture](snowplow.png)
 
-Architecutre diagram: https://miro.com/app/board/uXjVO5oHq_U=/
+Architecture diagram: https://miro.com/app/board/uXjVO5oHq_U=/
 
 ## Folder structure
-- the infrastructure code is present in `.aws`
-  - the SQS should be able to listen to the SNS/eventBridge rule that you want to onboard
 
-  
+- the infrastructure code is present in `.aws`
+  - the SQS should be able to listen to the SNS/eventBridge rule that you want to onboard.
 - the ECS application code is in `src`. This sends the event to snowplow.
-  - the `sqsConsumer` contains logic to consume from the SQS queue
-  - the `eventConsumer` contains logic to consume from event-bridge and transform them to snowplow
-  - the `snowplow` folder contains handlers to send events to snowplow
-- `.docker` contains local setup
-- `.circleci` contains circleCI setup
+  - the `sqsConsumer` contains logic to consume from the SQS queue.
+  - the `eventConsumer` contains logic to consume from event-bridge and transform them to snowplow.
+  - the `snowplow` folder contains handlers to send events to snowplow.
+- `.docker` contains local setup.
+- `.circleci` contains circleCI setup.
 
 ## How to onboard a new event?
+
 - in `.aws`, add the SNS arn to `SNSTopicsSubscriptionList`
 - in `eventConsumer`, please map the event-bridge payload to your snowplow format.
-- in `snowplow` folder, please construct your snowplow payload as per your schema model
-  - Make sure the schema version in `config.schema` that matches with dev/prod snowplow version
+- in `snowplow` folder, please construct your snowplow payload as per your schema model.
+  - Make sure the schema version in `config.schema` that matches with dev/prod snowplow version.
```

Comment:
this now gets added to the individual types file so we don't have conflicting schema